service_profiles.md

```diff
@@ -0,0 +1,111 @@
+# Service Profiles
+
+The RFC introduces the notion of service profiles.  A service profile is an
+object that describes characteristics that a service may have.  Throughout
+this RFC we will extensively use the REST Service Profile as an example.  The
```

Comment:
Maybe this is overly pedantic, but aren't the idempotency semantics around GETs vs POSTs vs whatever a function of the HTTP spec, not REST? If so, would it be more accurate to call this the "Conforming HTTP service profile" or something like that?